source/percona-release.rst

```diff
@@ -67,7 +67,7 @@ official package:
 
 #. Once you install this package the |Percona| repositories should be available. You
    can check the repository setup in the
-   :file:`/etc/apt/sources.list.d/percona-release.list` file.
+   :file:`/etc/apt/sources.list.d/percona-original-release.list` file.
```

Comment:
it is only about original repository.
if user will enable any other repo - the file name will be different